### DISCUSSION TITLE
Report on the use of application v2.3.1 on S6E

### DISCUSSION THREAD

**Discussion by maodun96** [seaky/nspanel_pro_tools_apk#159] [General]
I tested nspanel_pro_tools_v2.3.1 on S6E. The following is my report content.

new features

- Zigbee gateway integration✖

-  - possibility to change between router and coordinator mode✖
> The S6E is different from the nspanel pro, which includes a built-in Mosquitto broker and Zigbee2MQTT. 
Even if the Mosquitto broker is deployed, it may be missing some critical configurations.
so it's possible that this feature not working is normal. The same may apply to other non-nspanel pro devices.
-  - custom firmware feature still not available 
> If the firmware referred to here is for the Zigbee chip, then non-nspanel pro devices should not have any expectations regarding this.
The Zigbee chip model for the S6E is EFR32MG13 or TYZS13 (produced by EFR32MG13), while the Zigbee chip model for the nspanel pro is EFR32MG21.
I have attempted to update the Zigbee firmware on one of my S6Es, but many more S6Es cannot be updated. For details, please refer to the :https://github.com/grobasoz/zigbee-firmware/issues/52
- Wake on network error. If network error occurs it assits to the OS to reconnect.✔
- Switch to default app via MQTT✔

--------------------------------------------------------------
- Remote / MQTT app configuration
- Set sleep time via MQTT

> No testing was performed

**Comment by seaky** [comment]
many thanks for the report!

**Reply by maodun96** [reply]
Is there any way to enable the Zigbee functionality on the S6E using software? 
For example, could the Mosquitto broker from the nspanel pro be fully ported to the S6E? 
I am really eager to use the device as a repeater.😂

**Reply by seaky** [reply]
It really depends on the installed firmware on the efr chip. On nspanel there is an emberznet 6.7 so it depends on that. What I implemeted the portable binaries (or stack) because it tends to change beteween sonoff fw versions.
So the answer is technically yes

**Reply by maodun96** [reply]
> It really depends on the installed firmware on the efr chip. On nspanel there is an emberznet 6.7 so it depends on that. What I implemeted the portable binaries (or stack) because it tends to change beteween sonoff fw versions. So the answer is technically yes

I understand, and it's quite similar to what I guessed. 

Since it's done by replacing the firmware, and my S6E cannot be freely updated (using the S6E or T6E from TYZS13 might have some possibility), I can only give up on it. 

Thank you again for developing this application and for your patient guidance. 

Regards.
